### ISSUE TITLE
Captions of `:float multicolumn` figures not exported with ox-word 

### ISSUE THREAD

**Issue by dangom** [jkitchin/scimax#271]
I noticed that ox-word exported files are missing the captions of all figures tagged with `#+attr_latex: :float multicolumn` (I suppose some issue related to the figure* environment).

My immediate solution was to try using a before-processing hook to flush all lines containing `attr_latex`, as in:

```elisp
(defun ox-word-flush-attr-latex (backend)
  (when (org-export-derived-backend-p backend 'MSWord)
    (flush-lines "attr_latex")))

(add-hook 'org-export-before-processing-hook 'ox-word-flush-attr-latex)
```

Unfortunately, although the MSWord exporter is defined as a derived backend, when the above `ox-word-flush-attr-latex` is called, the backend is shown as if it were `latex` (which makes sense, since `ox-export-via-latex-pandoc-to-docx-and-open` simply calls `org-latex-export-to-latex`).

**Comment by jkitchin** [comment]
can you post a small org file that shows the issue?

**Comment by dangom** [comment]
Of course.

```
#+TITLE: MWE

* This is an MWE

Integer placerat tristique nisl, as shown in ref:fig.

#+attr_latex: :float multicolumn
#+name: fig
#+caption: Legend will not show up in docx export.
[[./image.png]]

Nunc justo sit amet elit as shown in ref:fig2.

#+name: fig2
#+caption:  Legend will show up.
[[./image.png]]

Nunc porta vulputate tellus.

```

**Comment by jkitchin** [comment]
I guess this is an issue with pandoc not handling the figure* environment.  There probably is not much i can do to fix that.

I am unclear on what solution you are looking for. Do you just want to strip lines containing 

#+attr_latex: :float multicolumn

?

**Comment by dangom** [comment]
Yes, this was my idea. But then I didn't know how to do that cleanly since checking for a MSWord backend doesn't work, so hooks are not selective enough. I want the keep the float multicolumn when exporting to latex/pdf.

**Comment by jkitchin** [comment]
I would solve this with a local build block like:


```
* build                                                            :noexport:

#+BEGIN_SRC emacs-lisp
(let ((org-export-before-parsing-hook (lambda (backend)
					(while (re-search-forward ":float multicolumn" nil t)
					  (replace-match "")))))
  (ox-export-via-latex-pandoc-to-docx-and-open))

#+END_SRC

```

**Comment by jkitchin** [comment]
It is one of my favorite tricks! If there was already a setting for that hook, it gets restored after the let statement.
